spec/lexers/sml_spec.rb

```diff
@@ -1,23 +1,113 @@
-# # -*- coding: utf-8 -*- #
-
-# describe Rouge::Lexers::Sml do
-#   let(:subject) { Rouge::Lexers::Sml.new }
-# 
-#   describe 'guessing' do
-#     include Support::Guessing
-# 
-#     it 'guesses by filename' do
-#       assert_guess :filename => 'foo.???'
-#     end
-# 
-#     it 'guesses by mimetype' do
-#       assert_guess :mimetype => 'text/x-sml'
-#       assert_guess :mimetype => 'application/x-sml'
-#     end
-# 
-#     it 'guesses by source' do
-#       assert_guess :source => '????'
-#     end
-#   end
-# end
-# 
+# -*- coding: utf-8 -*- #
+
+describe Rouge::Lexers::SML do
+  let(:subject) { Rouge::Lexers::SML.new }
+
+  describe 'lexing' do
+    include Support::Lexing
+
+    it 'handles let..in..end' do
```

Comment:
We don't spec lexers this way because speccing to the exact tokens would be incredibly brittle. Instead, please add this sample to `spec/visual/samples/sml` and test visually by running `rackup` and visiting `localhost:9292`.